low-level/files/opt/seagate/sspl/conf/sspl.init.conf.tmpl

```diff
@@ -0,0 +1,47 @@
+cluster:
```

Comment:
This init stage config template should had all input configs which required by SSPL, basically the SSPL config super set.

Reply:
Missing keys are added.